website/js/index.js

```diff
@@ -448,6 +479,84 @@ var VariantDetail = React.createClass({
 				</tr>);
         });
 
+        var versionRows = [];
+        // which keys represent comma seperated lists? for filtering out re-ordered lists
+        var listKeys = [
+            "Submitter_ClinVar",
+            "Method_ClinVar",
+            "Source",
+            "Date_Last_Updated_ClinVar",
+            "Source_URL",
+            "SCV_ClinVar",
+            "Clinical_Significance_ClinVar",
+            "Allele_Origin_ClinVar"
+        ];
+        for (var i = 0; i < data.length; i++) {
+            let version = data[i],
+                changes = [],
+                release = version["Data_Release"], //eslint-disable-line dot-notation
+                hightlightRow = false;
+
+            // if this is not the oldest version of the variant, diff them
```

Comment:
im a little confused about the intention here, could you clarify?

im assuming it's supposed to diff the current version of a variant with whatever the previous version was and nothing more.